### PR TITLE
in azure cosmosdb create relationship with firewall rule

### PR DESCRIPTION
**Pull request by sagarnavale**

### CODE REVIEW

File: cartography/intel/azure/cosmosdb.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from typing import Generator
 from typing import List
 from typing import Tuple
-
+import ipaddress
```
Comment (mpurusottamc) [inline cartography/intel/azure/cosmosdb.py:8]: is this an external pypi package? if yes, we need to add it to requirements.txt file

**Review by mpurusottamc — COMMENTED** [review]
one clarification needed.

File: cartography/intel/azure/cosmosdb.py (hunk 1 above)
Comment (sagarnavale) [reply cartography/intel/azure/cosmosdb.py:8]: no need install this package.

**Review by mpurusottamc — APPROVED** [review]
looks good

**Pull request merged**